library/functions.po

```diff
@@ -307,70 +308,63 @@ msgid ":func:`round`"
 msgstr ":func:`round`"
 
 #: ../../library/functions.rst:45
-#, fuzzy
 msgid ""
 "Return the absolute value of a number.  The argument may be an integer or a "
 "floating point number.  If the argument is a complex number, its magnitude "
 "is returned."
 msgstr ""
-"返回一個數的絕對值。實參可以是整數或浮點數。如果實參是一個複數，返回它的模。"
+"返回一個數的絕對值。引數可以是整數或浮點數。如果引數是一個複數，返回它的純量"
+"(大小)。"
 
 #: ../../library/functions.rst:52
-#, fuzzy
 msgid ""
 "Return ``True`` if all elements of the *iterable* are true (or if the "
 "iterable is empty).  Equivalent to::"
 msgstr ""
-"如果 *iterable* 的所有元素為 True（或 iterable 為空），返回 ``True`` 。等價"
-"於::"
+"如果 *iterable* 的所有元素皆為 True（或 iterable 為空），返回 ``True`` 。等價"
+"於："
 
 #: ../../library/functions.rst:64
-#, fuzzy
 msgid ""
 "Return ``True`` if any element of the *iterable* is true.  If the iterable "
 "is empty, return ``False``.  Equivalent to::"
 msgstr ""
 "如果 *iterable* 的任一元素為 True，返回 ``True`` 。如果 iterable 為空，返回 "
-"``False``。等價於::"
+"``False``。等價於："
 
 #: ../../library/functions.rst:76
-#, fuzzy
 msgid ""
 "As :func:`repr`, return a string containing a printable representation of an "
 "object, but escape the non-ASCII characters in the string returned by :func:"
 "`repr` using ``\\x``, ``\\u`` or ``\\U`` escapes.  This generates a string "
 "similar to that returned by :func:`repr` in Python 2."
 msgstr ""
-"就像函式 :func:`repr`，返回一個物件可列印的字串，但是 :func:`repr` 返回的字串"
-"中非 ASCII 編碼的字元，會使用 ``\\x``、``\\u`` 和 ``\\U`` 來轉義。生成的字串"
-"和 Python 2 的 :func:`repr` 返回的結果相似。"
+"就像函式 :func:`repr`，返回一個表示物件的字串，但是 :func:`repr` 返回的字串中"
+"非 ASCII 編碼的字元，會使用 ``\\x``、``\\u`` 和 ``\\U`` 來取代。這個函式生成"
+"的字串和 Python 2 的 :func:`repr` 返回的結果相似。"
 
 #: ../../library/functions.rst:84
-#, fuzzy
 msgid ""
 "Convert an integer number to a binary string prefixed with \"0b\". The "
 "result is a valid Python expression. If *x* is not a Python :class:`int` "
 "object, it has to define an :meth:`__index__` method that returns an "
 "integer. Some examples:"
 msgstr ""
-"將一個整數轉變為一個字首為“0b”的二進位制字串。結果是一個合法的 Python 表示"
+"將一個整數轉變為一個前綴為 “0b”的二進位制字串。結果是一個合法的 Python 表示"
 "式。如果 *x* 不是 Python 的 :class:`int` 物件，那它需要定義 :meth:"
-"`__index__`  方法返回一個整數。一些例子："
+"`__index__` 方法返回一個整數。一些例子："
```

Comment:
It may sound better to use some more common way of saying "for example" in Chinese, such as "舉例來說".

Reply:
@adrianliaw , thanks for the detailed review.
Should I use "subclass" instead of "子類別"?
And, should all the words in the glossary having the "form xxxx（XXXX）" not be translated?
For examples, int（整數）、boolean（布林）、dictionary（字典）.
Or depends on the sentence?